miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/WebConfig.java

```diff
@@ -44,4 +53,18 @@ public RequestMappingHandlerAdapter requestMappingHandlerAdapter() {
     adapter.setWebBindingInitializer(bindingInitializer());
     return adapter;
   }
+
+  @Autowired
+  private List<BarcodableService<?>> barcodableServices;
+
+  @Bean
+  public Map<EntityType, BarcodableService<?>> barcodableServicesMap() {
+    return barcodableServices.stream().collect(Collectors.toMap(BarcodableService::getEntityType, Function.identity()));
+//    Map<EntityType, BarcodableService<?>> barcodableServicesMap = new HashMap<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)